react/testing_5/src/screens/Dashboard/components/SignupForm/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { Field, reduxForm } from 'redux-form'
+
+import CustomField from '../../../../components/Field';
+import { required } from './utils';
+
+function SignupForm(props) {
+  const { handleSubmit } = props
```

Comment:
semicolon `;`

Reply:
I'd destructure at param level to keep consistency